src/pybind/mgr/mgr_util.py

```diff
@@ -82,9 +82,17 @@ def format_bytes(n, width, colored=True):
 def merge_dicts(*args):
     # type: (dict) -> dict
     """
-    >>> assert merge_dicts({1:2}, {3:4}) == {1:2, 3:4}
-        You can also overwrite keys:
-    >>> assert merge_dicts({1:2}, {1:4}) == {1:4}
+    >>> merge_dicts({1:2}, {3:4})
+    {1: 2, 3: 4}
+
+    You can also overwrite keys:
+    >>> merge_dicts({1:2}, {1:4})
+    {1: 4}
+
+    This should fail:
+    >>> True
+    False
```

Comment:
yeah

Reply:
But it didn't fail because of that, seem like I shrunk the tox.ini to much.

Reply:
now it does!
```
mgr_util.py F                                                            [100%]

=================================== FAILURES ===================================
________________________ [doctest] mgr_util.merge_dicts ________________________
084 
085     >>> merge_dicts({1:2}, {3:4})
086     {1: 2, 3: 4}
087 
088     You can also overwrite keys:
089     >>> merge_dicts({1:2}, {1:4})
090     {1: 4}
091 
092     This should fail:
093     >>> True
Expected:
    False
Got:
    True
```

Reply:
Removed the temporary commit that made it fail.